test/ruby/test_objectspace.rb

```diff
@@ -94,7 +94,7 @@ def test_count_objects
   end
 
   def test_finalizer
-    assert_in_out_err(["-e", <<-END], "", %w(:ok :ok :ok :ok), [])
+    assert_in_out_err(["-e", <<-END], "", %w(:ok :ok :ok), [])
```

Comment:
My understanding is that this assertion was incorrect.

- We define 1 finalizer on the source object
- Clone the source object, so 1+1 => 2
- Add another finalizer on the copied object, so 2 + 1 => 3.